Main.py

```diff
@@ -0,0 +1,41 @@
+from Mitarbeiter import Mitarbeiter
+from Vorgesetzter import Vorgesetzter
+
+
+def main() -> None:
+    walterWinkelmann: Mitarbeiter = Mitarbeiter("Walter Winkelmann")
+    waltraudWichtig: Vorgesetzter = Vorgesetzter("Waltraud Wichtig")
+    hermannWichtiger: Vorgesetzter = Vorgesetzter("Hermann Wichtiger")
```

Comment:
```suggestion
    hermannWichtiger: Vorgesetzter = Vorgesetzter("Hermann Wichtiger")
    
```

Add some new lines in this file ;)